dist/alert-dialog/ds4/alert-dialog.css

```diff
@@ -32,9 +32,9 @@
   text-align: center;
 }
 .alert-dialog__title {
-  font-size: 1.25rem;
+  font-size: 1.5rem;
```

Comment:
Seeing changes to dialog in the PR which probably shouldn't be present...? Unless it's a design wide change to typography?